add PureScript 0.15.14

peterbecich opened the pull request:
SHAs retrieved from https://github.com/purescript/purescript/releases/tag/v0.15.14

peterbecich committed:
add PureScript 0.15.14

SHAs retrieved from https://github.com/purescript/purescript/releases/tag/v0.15.14

ursi commented:
I get this when I run `nix flake check`
```
error: hash '393a8955b40e945095f469a976421f8dfa595040' has wrong length for hash type 'sha256'
```

peterbecich committed:
correct SHAs

```
$ nix-prefetch-url https://github.com/purescript/purescript/releases/download/v0.15.14/linux64.tar.gz
0i717gb4d21m0pi1k90g5diq3yja1pwlw6ripv0d70jdnd9gsdl9

$ nix-prefetch-url https://github.com/purescript/purescript/releases/download/v0.15.14/macos.tar.gz
01973wiybblfbgjbqrhr8435y6jk6c94i667nr3zxkxy4np3lv3q
```

peterbecich commented:
good catch, thanks.

`nix flake check` succeeds now.

SHAs from:
```
$ nix-prefetch-url https://github.com/purescript/purescript/releases/download/v0.15.14/linux64.tar.gz
0i717gb4d21m0pi1k90g5diq3yja1pwlw6ripv0d70jdnd9gsdl9

$ nix-prefetch-url https://github.com/purescript/purescript/releases/download/v0.15.14/macos.tar.gz
01973wiybblfbgjbqrhr8435y6jk6c94i667nr3zxkxy4np3lv3q
```

ursi commented:
thanks!
